docs/app/components/ButtonDocs/props-default.txt

```diff
@@ -0,0 +1,7 @@
+defaultProps: {
+  disabled: false,
+  fullWidth: false,
+  mobileFullWidth: false,
+  outline: false,
+  type: 'primary'
+}
```

Comment:
Once you remove the unnecessary false default props from the component make sure to update these props as well.